src/components/manners/implementation/base.ts

```diff
@@ -9,6 +11,17 @@ export function implementation(opts: ImplementationOptions): Implementation {
     log: opts.configuration.debug ? console.log : () => { },
     warn: opts.configuration.debug ? console.warn : () => { },
 
-    wnfsWasmLookup: wnfsVersion => fetch(`https://unpkg.com/wnfs@${wnfsVersion}/wasm_wnfs_bg.wasm`)
+    // WASM
+    wnfsWasmLookup: wnfsVersion => fetch(`https://unpkg.com/wnfs@${wnfsVersion}/wasm_wnfs_bg.wasm`),
+
+    // File system
+    fileSystem: {
+      hooks: {
+        afterLoadExisting: (fs: FileSystem.API) => addSampleData(fs),
```

Comment:
A couple of questions here. First, should this function be `async`?

```suggestion
        afterLoadExisting: async (fs: FileSystem.API) => addSampleData(fs),
```

Second, not sure about adding sample data for every app. In product discussions with Jeff, we decided to remove the sample data for app-owned apps with the idea that they should control what data is created. Defaults won't really make sense in all cases.

Reply:
That function is `async` yeah, `addSampleData` returns a Promise.
Yeah we can remove the default, could use it as a hook for Fission Drive now 👍